pyscitt/pyscitt/client.py

```diff
@@ -313,29 +313,31 @@ def get_historical(self, *args, retry_on=[], **kwargs):
             retry_on=[
                 HTTPStatus.ACCEPTED,
                 (HTTPStatus.NOT_FOUND, "TransactionPendingOrUnknown"),
```

Comment:
Do we still need these two cases?

Reply:
The prefix tree code was still using Accepted for this. I've switched it to ServiceUnavailable to be consistent and removed these.